contracts/strategy/BaseStrategy.sol

```diff
@@ -117,26 +116,18 @@ abstract contract BaseStrategy is IStrategy, AccessControl {
         vault = _vault;
         underlying = IVault(_vault).underlying();
         ethAnchorRouter = IEthAnchorRouter(_ethAnchorRouter);
-        exchangeRateFeeder = IExchangeRateFeeder(_exchangeRateFeeder);
+        aUstToUstFeed = _aUstToUstFeed;
         ustToken = _ustToken;
         aUstToken = _aUstToken;
         perfFeePct = _perfFeePct;
 
+        _aUstToUstFeedDecimals = 10**_aUstToUstFeed.decimals();
```

Comment:
this is not decimals. decimals would be `18`, but this variable is `10**18`

Reply:
but i couldn't find best name for that. any suggestions?

Reply:
how about just calling `decimals()` here, and doing the `10**` down there, where the calculation is done?

Reply:
decimals is a constant value, i don't like spend gas to get decimals every time.

Reply:
then how about `_aUstToUstFeedMultiplier`?

Reply:
fixed